fractal_server/app/api/v1/workflow.py

```diff
@@ -65,7 +65,7 @@ async def create_workflow(
     return db_workflow
 
 
-@router.delete("/{_id}", status_code=status.HTTP_204_NO_CONTENT)
+@router.delete("/{_id}/", status_code=status.HTTP_204_NO_CONTENT)
```

Comment:
please remove trailing '/' after path parameters.
The slash should only appear after a constant string, such as "/{_id}/add-task/", as this way we avoid a few name clashes among endpoints.

I.e.,
`/{_id}/` and `/do-something/` both match the same pattern `/*/`, but `/{_id}` and `/do-something/` do not